### ISSUE TITLE
is there any plan to add semantic segmentation of Lidar to the opensource version?

### ISSUE THREAD

**Issue by arminiusresistance** [xtreme1-io/xtreme1#276]
Hi, 
for my project, I am looking for an open source project. 
I am an academic researcher and not affiliated with other companies which I don't own.

So, it would be good to have this tool for research and non-commercial purposes.

**Comment by jaggerwang** [comment]
Yes, we have plan to migrate semantic segmentation tool of cloud version to open source version, but there is currently no manpower available, may be a few months later.

**Comment by timohl** [comment]
Do you have any update on the migration of the semantic segmentation tool for point clouds to open source?

Yesterday I got an email from basic.ai marketing stating: 
> we have decided to discontinue the SaaS version of BasicAI Cloud by 31st October 2024

So, free basic.ai seems to be dead and private cloud is available:
> The pricing for private deployment starts at $2,400 per year

I am a student working on my master thesis and started to use basic.ai due to the nice semantic segmentation tool for point clouds.
Now with the free version gone and not having a budget for private deployment due to being a student, I have to look for alternatives.
So, I would love to use the open source version, but semantic segmentation is the big missing feature for my application.

**Comment by jaggerwang** [comment]
Thanks for your attention. Due to team shortage, new feature development has been suspended.

**Comment by zwtgit** [comment]
看起来是有了市场以后全面进行商业化了，这个项目仓库只是引流的项目

**Comment by ClementLeBihan** [comment]
Hi, Any update ?

**Comment by yekruz** [comment]
@jaggerwang 
Hi is there any updates on this?

**Comment by AbhiKora** [comment]
Hello, is there any update on the addition of segmentation tools to open source version?
